Update editor.c

soumyarup-paul opened the pull request:
Added Left/Right traversal using A/D keys(can be changed later)!In my next update i might use system calls to record keystrokes else the linked list is recording some extra enter keys(i'll try to find an alternative though)!

soumyarup-paul committed:
Update editor.c

Added Left/Right traversal using A/D keys(can be changed later)!In my next update i might use system calls to record keystrokes else the linked list is recording some extra enter keys(i'll try to find an alternative though)!

jayantbh reviewed (commented):
Wait, what? Why are you doing anything to do with cursor actions here? O.o

soumyarup-paul committed:
Update editor.c

Changed variable names for ease of understanding,removed the extra undesired NewLine character that was getting stored in the file.
This code alone can be used as a file editor by traversing a file left/right using A/D keys respectively(this will be deleted on the later versions when interfaced with GUI.I added this for the Backend dev team to try out the editor as a standalone before interfacing with the GUI).

soumyarup-paul committed:
Update editor.c

Fixed New File Creation Glitch!

soumyarup-paul committed:
Update editor.c

Finally!!! Deleted all the extra undesired NewLine characters! YaY!!

soumyarup-paul committed:
Update editor.c

Added 2 APIs for the front end

jayantbh commented:
Please, squash/amend commits for minor changes, and write more descriptive commit messages otherwise.
